bin/plot-constellations.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+usage() {
+  echo -n "##################################################
+# Make a png with an RA/DEC grid overlay from a FITS image and a PPM input file.
+#
+# Uses imagemagick tool to flip jpeg image and astrometry.net 'plot-consellations' tool to create overlay.
+#
+# If present the TITLE is added as a title to the png.
+##################################################
+ $ $(basename $0) FILENAME PPM
+
+ Options:
+  FILENAME          Name of fits file to be converted into a png overlay.
+  JPEG               JPEG input file to get image pixels (will get converted to PPM).
+
+ Example:
+  plot-constellations Documents/pole_canon01.fits pole_canon01.jpg 
+"
+}
+
+if [ $# -eq 0 ]; then
+    usage
+    exit 1
+fi
+
+export PATH=/usr/share/doc:$PATH
```

Comment:
This line does not work. I'm not sure how to add the `imagemagick` installation to my path.